src/wx/config/option-id.h

```diff
@@ -112,6 +111,7 @@ enum class OptionID {
     kGeomWindowY,
 
     /// UI
+    kUIEnableStatusBar,
```

Comment:
While I agree that this is a better place for it, this has the unfortunate side effect of breaking the setting for existing users.
There is no currently no mechanism to rename an existing INI option. It is something I would like to do down the line and do a (hopefully one-time) rename of options to have more consistent naming. In the meantime, I'm going to ask you leave this to `kGenStatusBar` so we don't break existing users.

Reply:
Thanks for the comment, Steelskin. I'll change `kUIEnableStatusBar` and `EnableStatusBar` back to `kGenStatusBar` and `StatusBar`.